src/main/java/thrift/logic/commands/UntagCommand.java

```diff
@@ -0,0 +1,130 @@
+package thrift.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import thrift.commons.core.Messages;
+import thrift.commons.core.index.Index;
+import thrift.logic.commands.exceptions.CommandException;
+import thrift.logic.parser.CliSyntax;
+import thrift.model.Model;
+import thrift.model.tag.Tag;
+import thrift.model.transaction.Description;
+import thrift.model.transaction.Expense;
+import thrift.model.transaction.Income;
+import thrift.model.transaction.Remark;
+import thrift.model.transaction.Transaction;
+import thrift.model.transaction.TransactionDate;
+import thrift.model.transaction.Value;
+
+/**
+ * Tags a specified Transaction
+ */
+public class UntagCommand extends Command {
+
+    public static final String COMMAND_WORD = "untag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Untags the transaction identified "
+            + "by the index number used in the displayed transaction list. "
+            + "The tags will be removed from existing tags, if they exist.\n"
+            + "Parameters: " + CliSyntax.PREFIX_INDEX + "INDEX (must be a positive integer) "
+            + CliSyntax.PREFIX_TAG + "TAG...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + CliSyntax.PREFIX_INDEX + "1 "
+            + CliSyntax.PREFIX_TAG + "Food";
+
+    public static final String MESSAGE_UNTAG_TRANSACTION_SUCCESS = "Updated Transaction: %1$s";
+    public static final String MESSAGE_TAG_NOT_EXISTED = "\nTag(s) %1$s do not exist and will be ignored.";
+    public static final String MESSAGE_ORIGINAL_TRANSACTION = "\n\nOriginal: %1$s";
+    public static final String MESSAGE_NOT_UNTAGGED = "At least one tag must be provided.";
+    public static final String MESSAGE_NO_DEL_TAGS = "Specified tags do not exist, the transaction was not updated.";
+
+    private final Index index;
+    private final Set<Tag> tagSet;
+    private final StringBuilder nonexistentTags;
+
+    /**
+     * Creates a UntagCommand to untag the specified {@code Transaction}
+     * @param index of the transaction in the filtered transaction list to update
```

Comment:
Do leave an empty line between description and parameter section.

Reply:
Added the empty line.